src/prefect/client/orion.py

```diff
@@ -794,13 +794,13 @@ async def read_work_queues(
 
     async def match_work_queues(
         self,
-        prefix: str,
+        prefixes: List[str],
```

Comment:
I think that it is less likely that anyone is using this interface directly and am okay with a breaking change here. Please keep an eye out of interface changes in the future though.

Reply:
Agreed on this case.